squidb-ios/src/com/yahoo/squidb/ios/IOSOpenHelper.java

```diff
@@ -24,8 +25,8 @@
     @Weak private final SquidDatabase.OpenHelperDelegate delegate;
 
     public IOSOpenHelper(String path, String name, SquidDatabase.OpenHelperDelegate delegate,
-            int version) {
```

Comment:
@azubala let's do the same changes in `AndroidOpenHelper` so that we cover both in one PR. WDYT?

Reply:
@kmalmur sure thing!